.cirrus.yml

```diff
@@ -232,6 +245,27 @@ windows_server_2019_x64_stable_release_task:
 
 # Pipeline 9
 
+freebsd_x64_min_version_debug_task:
+  depends_on: preflight_check
+  <<: *IOX2_CONTAINER_FREEBSD_X64
+  env:
+    PATH: /root/.cargo/bin:$PATH
+    HOME: /root  # must be set manually to '/root' or 'rustup' will throw an error
+  setup_script:
+    - uname -a
+    - pkg install -y git llvm
+    - pw useradd testuser1
+    - pw useradd testuser2
+    - pw groupadd testgroup1
+    - pw groupadd testgroup2
+    - kldload mqueuefs
+    - mkdir -p /mnt/mqueue/
+    - mount -t mqueuefs null /mnt/mqueue/
+    - curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --profile minimal --no-modify-path
+    - rustup component add clippy rustfmt
```

Comment:
If we keep the MSRV on each platform this should be refactored to a template and used in both FreeBSD tasks